php-frontend/src/main/java/org/sonar/php/cfg/LiveVariablesAnalysis.java

```diff
@@ -238,7 +238,7 @@ boolean propagate(Map<CfgBlock, LiveVariables> liveVariablesPerBlock) {
     @Override
     public void visitAssignmentExpression(AssignmentExpressionTree tree) {
       boolean isCompoundAssignment = !tree.is(Tree.Kind.ASSIGNMENT, Tree.Kind.NULL_COALESCING_ASSIGNMENT);
-      boolean isUsedAsNamedParameterOrReturn = tree.getParent().is(Tree.Kind.FUNCTION_CALL, Tree.Kind.RETURN_STATEMENT);
+      boolean isUsedAsNamedParameterOrReturn = tree.getParent().is(Tree.Kind.FUNCTION_CALL_ARGUMENT, Tree.Kind.RETURN_STATEMENT);
```

Comment:
That's a kind a backward-incompatible change which is not in the API. I'm not sure whether we can document it.

Reply:
Yes, indeed. If one used to check if something is an argument by checking if the parent is a `FUNCTION_CALL`, it has to be adapted. I can add a line about it in the readme.